src/transaction.js

```diff
@@ -8,10 +9,12 @@ var Promise = require('bluebird');
 /**
  * @param {string} id
  * @param {Transaction~callback} callback
+ * @param {number} [timeout] in milliseconds. Timeout until transaction is executed. Default 30seconds.
```

Comment:
Instead of sentence docu, let's use jsdoc (http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values):
```js
@param {number} [timeout=30000]
```
agree?